README.md

```diff
@@ -35,12 +35,19 @@ This entire workflow supports the UX that users have come to expect from contain
 
 To support this UX the OCI Image Format contains sufficient information to launch the application on the target platform (e.g. command, arguments, environment variables, etc).
 
-## FAQ
+## Additional Artifact Types
+
+The OCI image spec provides the core elements required for additional artifact types, including [Helm](https://helm.sh/), [CNAB](https://cnab.io) and existing artifacts like Azure Resource Manager (ARM) or AWS Cloud Formation Templates. By utilizing the image-spec core elements, additional tool authors and registry implementors can easily support new artifact types, benefiting from the common infrastructure to deploy container images. 
```

Comment:
Nit: CloudFormation is misspelled (no space), but also it seems odd to call this out as an example in the official spec when I'm not aware of any plans for us (AWS) to add this as a media type in the near term.  Will that be a community contribution? :)

Reply:
Hey Clare, 
I was just looking for examples to demonstrate the range of ideas. If you prefer I remove, I can do that. I've been thinking of doing the doc artifactType as a concrete example, in addition to helm. Just let me know what you prefer.

Reply:
Per Clare: removing as the work isn't currently planned, but very interested in the feature.